src/cnd/axios_rfc7807_middleware.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { Problem } from "./axios_rfc7807_middleware";
+
+describe("problem message", () => {
+  it("should format all fields to a descriptive message", () => {
+    const problem = new Problem({
+      title: "Swap not supported.",
+      status: 400,
+      detail: "This combination of protocols is not supported.",
+      type: "https://comit.network/docs/errors/swap-not-supported"
```

Comment:
One day, I hope that we can document all those errors as part of our docs and fill those URLs into the error responses of cnd to present users with beautiful, helpful messages <3

Reply:
I would like that too, but for now I would only add links that point to something that exists already.
If a dev receives such an error and tries to use the URL it will cause more confusion than it helps at the moment.

Reply:
Would it always exclude that at the moment because the URL is not present?

Reply:
> Would it always exclude that at the moment because the URL is not present?

If the URL is not present in the response, nothing in regards to the URL is part of the output.

The response comes from `cnd` so we have to start returning them there once they exist. The SDK never "assembles" these links (check the code on how this message is generated)!